Source/Scene/PointCloud3DTileContent.js

```diff
@@ -91,9 +91,22 @@ define([
         this._pickCommand = undefined;
         this._pickId = undefined; // Only defined when batchTable is undefined
         this._isTranslucent = false;
+        this._styleTranslucent = false;
         this._constantColor = Color.clone(Color.WHITE);
         this._rtcCenter = undefined;
 
+        // These values are used to regenerate the shader when the style changes
+        this._styleableShaderAttributes = undefined;
+        this._isQuantized = false;
+        this._isOctEncoded16P = false;
+        this._hasColors = false;
+        this._hasNormals = false;
+        this._hasBatchIds = false;
+
+        // TODO : How to expose this? Will this be part of the point cloud styling or a property of the tileset?
```

Comment:
Doesn't need to be part of this PR; we can determine this when we also consider point attenuation, blurring, etc., which will have the same questions.